examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java

```diff
@@ -86,14 +86,35 @@ public static void setUp() {
   }
 
   @Test
-  public void testWindowedWordCountInBatch() throws Exception {
-    testWindowedWordCountPipeline(defaultOptions());
+  public void testWindowedWordCountInBatchDynamicSharding() throws Exception {
+    WindowedWordCountITOptions options = batchOptions();
+    // This is the default value, but make it explicit
+    options.setNumShards(null);
+    testWindowedWordCountPipeline(options);
+  }
+
+  @Test
+  public void testWindowedWordCountInBatchStaticSharding() throws Exception {
+    WindowedWordCountITOptions options = batchOptions();
+    options.setNumShards(3);
+    testWindowedWordCountPipeline(options);
+  }
+
+  @Test
```

Comment:
I think we should remove this test until we fix the streaming runner to not generate a file per bundle. Until then this is not a suggested use case for streaming, so I think it's ok to remove the test.